### PR TITLE
Feat/upgrade

### PR DESCRIPTION
**Pull request by NotMaharshi**
## what
* Removed deprecations and errors
* Replaced deprecated arguments with new ones
* Updated CODEOWNERS
* Updated github actions workflows
* Renamed directory _examples to example
* Added new formatting for subscription id

## why
* Schedule maintenance

### CODE REVIEW

File: examples/linux-vm/example.tf [hunk 1]
```diff
@@ -1,5 +1,11 @@
 provider "azurerm" {
   features {}
+  subscription_id = "01111111111110-11-11-11-11"
```
Comment (13archit) [inline examples/linux-vm/example.tf:3]: Replace it with
`"000000-11111-1223-XXX-XXXXXXXXXXXX"`

File: examples/linux-vm/example.tf [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+}
+provider "azurerm" {
+  features {}
+  alias           = "peer"
+  subscription_id = "01111111111110-11-11-11-11"
```
Comment (13archit) [inline examples/linux-vm/example.tf:8]: Replace it with
`"000000-11111-1223-XXX-XXXXXXXXXXXX"`

File: examples/windows-vm/example.tf [hunk 3]
```diff
@@ -1,5 +1,13 @@
 provider "azurerm" {
   features {}
+  subscription_id = "01111111111110-11-11-11-11"
```
Comment (13archit) [inline examples/windows-vm/example.tf:3]: Replace it with 
`"000000-11111-1223-XXX-XXXXXXXXXXXX"`

File: examples/linux-vm/example.tf (hunk 2 above)
Comment (NotMaharshi) [reply examples/linux-vm/example.tf]: its done

File: examples/windows-vm/example.tf (hunk 3 above)
Comment (NotMaharshi) [reply examples/windows-vm/example.tf]: its done

**Review by 13archit — APPROVED** [review]
LGTM

**Pull request merged**